lib/Sat/CryptoMinisat5.cpp

```diff
@@ -169,7 +169,7 @@ uint32_t CryptoMiniSat5::getFixedCountWithAssumptions(const stp::SATSolver::vec_
 
   //std::cerr << assumps.size() << " assumptions" << std::endl;
 
-  unsigned assigned = 0;
+  uint32_t assigned = 0;
```

Comment:
For consistency with the rest of the code, I'd make this `unsigned int` (adding `uint32_t` makes it seem like we really know it is 32-bit limited).

Reply:
I used `uint32_t` only because the return type of the function is marked as `uint32_t` and that value is returned from it.

Reply:
GOOD SPOT!!! You're totally correct!!!